### ISSUE TITLE
Redesign AreaOfStudy component

### ISSUE THREAD

**Issue by hawkrives** [hawkrives/gobbldygook#815]
I have a mockup; I need to scan it in.

**Comment by hawkrives** [comment]
Why did I not scan this in.

**Comment by hawkrives** [comment]
I'm going to close it; we know that they need to be redesigned (even after #2299, but it's less pressing now) and so it seems less like a single task and more like a number of incremental ones.

If we come up with a giant idea, we can open a new issue.
